File: iCyption2022/CftTest/stand/solution.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
  int n;
  cin >> n;
  int harga[n];
  for (int i = 0; i < n; i++) {
    cin >> harga[i];
  }
  int totalsewa[n];
  totalsewa[0]=0;
  if (n>0) {
    totalsewa[1]=harga[0];
    for (int i = 2; i < n+1; i++){
      totalsewa[i] = max(totalsewa[i-1],harga[i-1]+totalsewa[i-2]);
    }
  }
  cout << totalsewa[n] << endl;
}

File: iCyption2022/CftTest/stand/solution.py
n = int(input())
if (n == 0):
    print('0')
else:
    harga = list(map(int, input().split()))

    totalSewa = [0 for i in range(n + 1)]

    totalSewa[0] = 0
    totalSewa[1] = harga[0]
    for i in range(2, n + 1):
        totalSewa[i] = max(totalSewa[i - 1], harga[i - 1] + totalSewa[i - 2])
    print(totalSewa[n])

